### PR TITLE
(Post) Add team update for 2020-03-13

### PR DESCRIPTION
**Pull request by carabasdaniel**

### CODE REVIEW

File: _posts/2020-03-13-status-update.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+## Updates to Puppet Modules
+
+Over the last week we have released:
+
+- [puppetlabs/apt](https://forge.puppet.com/puppetlabs/apt) (v7.4.1)
+
+## Community
+
+A big thank you goes out to Glen Sarti for the work he has done in removing the PDK dependecy from Litmus. Also we would like to thank [Antoine](https://github.com/anarcat) and [Dirk Heinrichs]() for their help in resolving the file permission issues on the [puppetlabs-apt](https://github.com/puppetlabs/puppetlabs-apt) module.
```
Comment (DavidS) [inline _posts/2020-03-13-status-update.md]:
```suggestion
A big thank you goes out to [Glenn Sarti](https://github.com/glennsarti) for the work he has done in removing the PDK dependency from Litmus.
Also we would like to thank [Antoine](https://github.com/anarcat) and [Dirk Heinrichs](https://tickets.puppetlabs.com/secure/ViewProfile.jspa?name=dhs%40recommind.com&selectedTab=com.atlassian.streams.streams-jira-plugin:user-profile-stream-panel) for their help in resolving the file permission issues on the [puppetlabs-apt](https://github.com/puppetlabs/puppetlabs-apt) module.
```
* link to Glenn's profile
* link to Dirk's profile
* spell check
* semantic linebreaks

File: _posts/2020-03-13-status-update.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+## Litmus Progress
+
+We have release [Litmus version 0.16.0](https://github.com/puppetlabs/puppet_litmus/releases/tag/0.16.0) that contains a couple of nice fixes and removes dependency on PDK - [CHANGELOG](https://github.com/puppetlabs/puppet_litmus/blob/master/CHANGELOG.md)
+
+On [puppetlabs-vsphere](https://github.com/puppetlabs/puppetlabs-vsphere) we have finished the conversion to allow users to run acceptance tests using Litmus in [PR](https://github.com/puppetlabs/puppetlabs-vsphere/pull/172). 
+
+Finally the conversion for using Litmus for [puppetlabs-docker](https://github.com/puppetlabs/puppetlabs-docker) acceptance testing is ready to be merged in [PR](https://github.com/puppetlabs/puppetlabs-docker/pull/585). As a result any future pull request to the docker module will also run the acceptance tests using Github Actions hosted runners on Ubuntu 16.04 and Ubuntu 18.04. For running acceptance tests on Windows, the provision file of the module contains a item that uses a Windows Vagrant box for testing.
```
Comment (DavidS) [inline _posts/2020-03-13-status-update.md]:
```suggestion
Finally the conversion for using Litmus for [puppetlabs-docker](https://github.com/puppetlabs/puppetlabs-docker) acceptance testing is ready to be merged in [puppetlabs-docker#585](https://github.com/puppetlabs/puppetlabs-docker/pull/585).
As a result any future pull request to the docker module will also run the acceptance tests using Github Actions hosted runners on Ubuntu 16.04 and Ubuntu 18.04.
For running acceptance tests on Windows, the provision file of the module contains a item that uses a Windows Vagrant box for testing.
```

* semantic line breaks
* use github notation for the PR link text
* please add a link to GHA run for docker.

File: _posts/2020-03-13-status-update.md [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+
+## Customer Escalations
+
+Over the past week we have closed out two issues on the [puppetlabs-apt](https://github.com/puppetlabs/puppetlabs-apt) module. As a result of these two tickets [MODULES-10548](https://tickets.puppetlabs.com/browse/MODULES-10548) and [MODULES-10583](https://tickets.puppetlabs.com/browse/MODULES-10583) we have decided to remove the file permissions management from the module implementation.
+
+We have also merged a PR that will remove the strftime function from the [puppetlabs-stdlib](https://github.com/puppetlabs/puppetlabs-stdlib) module as this function has been deprecated and superseded by a function withing the Puppet agent, see [IAC-547](https://tickets.puppetlabs.com/browse/IAC-547).
+
+## Litmus Progress
+
+We have release [Litmus version 0.16.0](https://github.com/puppetlabs/puppet_litmus/releases/tag/0.16.0) that contains a couple of nice fixes and removes dependency on PDK - [CHANGELOG](https://github.com/puppetlabs/puppet_litmus/blob/master/CHANGELOG.md)
```
Comment (DavidS) [inline _posts/2020-03-13-status-update.md]:
```suggestion
We have released [Litmus version 0.16.0](https://github.com/puppetlabs/puppet_litmus/releases/tag/0.16.0) that contains a couple of nice fixes and removes dependency on PDK - [CHANGELOG](https://github.com/puppetlabs/puppet_litmus/blob/master/CHANGELOG.md)
```

File: _posts/2020-03-13-status-update.md [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+
+On [puppetlabs-vsphere](https://github.com/puppetlabs/puppetlabs-vsphere) we have finished the conversion to allow users to run acceptance tests using Litmus in [PR](https://github.com/puppetlabs/puppetlabs-vsphere/pull/172).
```
Comment (DavidS) [inline _posts/2020-03-13-status-update.md]:
```suggestion
On [puppetlabs-vsphere](https://github.com/puppetlabs/puppetlabs-vsphere) we have finished the conversion to allow users to run acceptance tests using Litmus in [puppetlabs-vsphere#172](https://github.com/puppetlabs/puppetlabs-vsphere/pull/172). 
```

File: _posts/2020-03-13-status-update.md (hunk 2 above)
Comment (carabasdaniel) [reply _posts/2020-03-13-status-update.md]: Can add link to GHA, but it is on my fork, will run only after merge on push or PR

Comment (DavidS) [reply _posts/2020-03-13-status-update.md]: nevermind then.

**Comment by DavidS** [comment]
>     * rook release might go out by tomorrow?
this is happening later tonight, since it's only a small release we don't have to hold the post

>     * filter improvement on community reports (anything else there?)
If you don't want to say something about this, we can merge&publish.

**Pull request merged**